### ISSUE TITLE
Search using diacritics does not highlight word in autocomplete results

### ISSUE THREAD

**Issue by tfrancart** [NatLibFi/Skosmos#423]
Hello

I tried skosmos 1.4 + fuseki-2.4.0-20160102.095717-24. I used the jena-text configuration file template provided in the documentation, with extra parameters for skosmos 1.4, and tried the following text analyzer configuration :

           text:analyzer [
             a text:ConfigurableAnalyzer ;
             text:tokenizer text:LetterTokenizer ;
             text:filters (text:ASCIIFoldingFilter text:LowerCaseFilter)
           ] 

It works (nearly) fine, and allows to search inside labels, and without diacritics.
But when the search - without diacritic - matches a label that contain a diacritic, the corresponding word is not highlighted. See screenshot below where "Accès à l'éducation" is matched but not highlighted. This is a minor issue and I can live with it.

Thanks !

![accented-characters-not-bold](https://cloud.githubusercontent.com/assets/2728945/12091333/9205a8fc-b2f6-11e5-9c82-e1156b032433.png)

**Comment by osma** [comment]
Right. The highlighting in this case is performed by the typeahead.js widget, which AFAIK doesn't know anything about accent folding. I don't think we can do much.

**Comment by osma** [comment]
The relevant typeahead.js code is here:
https://github.com/twitter/typeahead.js/blob/master/src/typeahead/highlight.js

The highlighting is performed by applying a regex on the text. I don't know of any simple way in which accent folding could be incorporated into this mechanism.

**Comment by osma** [comment]
Closing and archiving issue due to lack of activity. Please reopen if there are concrete reasons for reconsidering.
